examples/load-torchscript/Main.hs

```diff
@@ -0,0 +1,33 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE RecordWildCards #-}
+
+module Main where
+
+import Torch.Script
+import Torch.Utils.Image
+import qualified Torch.DType                   as D
+import qualified Torch.Tensor                  as D
+import qualified Torch.Functional              as D
+
+import Control.Exception.Safe (catch,throwIO)
+import Language.C.Inline.Cpp.Exceptions (CppException(..))
+
+prettyException :: IO a -> IO a
+prettyException func =
+  func `catch` \a@(CppStdException message) -> do
+    putStrLn message
+    throwIO (CppStdException "")
+
+main :: IO ()
+main = prettyException $ do
+  model <- load "resnet_model.pt"
+  mimg <- readImage "elephant.jpg"
+  case mimg of
+    Left err -> print err
+    Right img'-> do
+      let img = IVTensor $ D.toType D.Float $ hwc2chw img'
+      v <- forward model [img]
+      case v of
+        IVTensor v' -> print $ D.argmax (D.Dim 1) D.RemoveDim v'
+        _ -> print "Return value is not tensor."
```

Comment:
resnet works, but maskrcnn throws this exception (https://gist.github.com/junjihashimoto/3326d9a45deb6419b5fba81c7ffa0c27) .

The exception may be the same as  one of this discussion. 
https://discuss.pytorch.org/t/fasterrcnn-resnet50-jit-trace/62337/9

Reply:
https://github.com/pytorch/vision/pull/1407
https://github.com/pytorch/vision/blob/v0.5.0/torchvision/csrc/vision.cpp#L48

Reply:
I'v realized what we need to use maskrcnn on haskell .

Reply:
Wow nice, you're able to run resnet inferences in Haskell?

Reply:
Yes.  The inferences work as follows. It says a spatula.(Input image is a elephant. )

```
$ stack run load-torchscript
Tensor Int64 [1] [ 831]
```

Now I can run maskrcnn too in Haskell by registering some ops.
https://github.com/pytorch/vision/pull/1843

Reply:
I've added some ivalue-types.
maskrcnn is available on haskell.

Reply:
Oh wow, I didn't see that you were working on this again. This is awesome!